tests/validator/test_validation_graph.py

```diff
@@ -185,3 +197,279 @@ def test_ExpectationValidationGraph_get_exception_info(
 
     assert left_exception in exception_info
     assert right_exception in exception_info
+
+
+@pytest.mark.integration
+def test_parse_validation_graph():
+    class PandasExecutionEngineStub:
+        pass
+
+    PandasExecutionEngineStub.__name__ = "PandasExecutionEngine"
```

Comment:
Why rename?

Reply:
@billdirks We need the engine to be Pandas for the logic of the test to work properly -- so this stubbing achieves the purposes without instantiating the actual `PandasExecutionEngine` class.  Thanks.